### ISSUE TITLE
High Training Loss (~1.5) with lycoris.kohya and algo=lokr

### ISSUE THREAD

**Issue by beiwilliam** [kohya-ss/musubi-tuner#325]
Hi everyone,

I'm encountering a very high training loss (around 1.5) when trying to train a LoKr model using the following settings:

network_module: lycoris.kohya
algo: lokr
I'm trying to train Wan's LoKr. Could anyone provide guidance on how to correctly configure the training for this setup or point out potential reasons for such high loss?

`--sdpa --mixed_precision bf16 --fp8_base --fp8_scaled --optimizer_type "adamw8bit" --learning_rate 1e-6  --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module lycoris.kohya --network_dim=1000000 --network_args "preset=full" "factor=8" "rank_dropout=0" "bypass_mode=False" "dora_wd=False" "module_dropout=0" "algo=lokr" "train_norm=False" --timestep_sampling shift --discrete_flow_shift 5.0 --max_train_steps 2500 --save_every_n_steps 150 --seed 2025`

**Comment by xianienie** [comment]
same problem   network_module: lycoris.kohya  algo: lokr

![Image](https://github.com/user-attachments/assets/7ddbf8b9-a3d0-408a-8b26-baa4b99d42bc)
Although the loss is decreasing, pure noise is generated.
